Gruntfile.js

```diff
@@ -25,6 +25,7 @@ module.exports = exports = function(grunt) {
             'src/utils/serializer': 'localforageSerializer',
             'src/drivers/indexeddb': 'asyncStorage',
             'src/drivers/localstorage': 'localStorageWrapper',
+            'src/drivers/memory': 'memoryStorageWrapper',
```

Comment:
Or better yet make it something with "volatile memory" so it is absolutely clear this is not persistent!